package.json

```diff
@@ -8,6 +8,7 @@
     "start:watch": "NODE_ENV=development nodemon",
     "build": "NODE_ENV=production tsc && cp -r src/views dist/ && gulp static",
     "lint": "eslint 'src/**/*'",
```

Comment:
for consistency you could add `src/*.ts ` here

Reply:
..done!